PrebidMobile/Objc/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -197,7 +197,7 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
                 PBMORTBVideo * const nextVideo = nextImp.video;
                 
                 if (!self.adConfiguration.adConfiguration.isOriginalAPI) {
-                    nextVideo.playbackend = @(2);
+                    nextVideo.playbackend = @(1);
```

Comment:
The `playbackend` should be changed only for interstitial video. But this code changes it for Oustream video as well. 

Once  the `playbackend` is changed only for interstitial video - please update the unit tests respectively.